<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Initial Information Confirmation (haha it rhymes)</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>451</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Host:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="hostLine">
      <property name="text">
       <string>engf0001.cs.ucl.ac.uk</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Type MQTT host address here...</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="portLine">
      <property name="text">
       <string>1883</string>
      </property>
      <property name="placeholderText">
       <string>Type port number here...</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Keep Alive:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="keepaliveLine">
      <property name="text">
       <string>60</string>
      </property>
      <property name="placeholderText">
       <string>Type keep_alive in seconds here...</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Topic:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="topicLine">
      <property name="text">
       <string>bioreactor_sim/nofaults/telemetry/summary</string>
      </property>
      <property name="placeholderText">
       <string>Type subscribed topic here...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="isSimulatedCheckBox">
      <property name="text">
       <string>is simulated?</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="decisionBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::NoButton</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
